procedures/candlepin/branch.md.erb

```diff
@@ -0,0 +1,23 @@
+# Branch Packaging
+
+- [ ] Clone https://github.com/theforeman/candlepin-packaging
+- [ ] Check out most recent branch `rpm/< latest_version >`
+- [ ] Create a new branch for the version being added
+  - `git checkout -b rpm/<%= release %>`
+- [ ] Update `package_manifest.yaml` with <%= release %>
+- [ ] Create project in copr `obal copr-project candlepin-staging-<%= release %>`
+- [ ] Push new branch to https://github.com/theforeman/candlepin-packaging
+
+# Update CI
+
+- [ ] Open a PR with the following updates:
+  - [ ] Add new vars file for the version being branched, [for example](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/vars/candlepin/4.4.groovy)
+  - [ ] Add <%= release %> to https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/yaml/includes/candlepin_versions.yaml.inc
+  - [ ] Add <%= release %> to https://github.com/theforeman/jenkins-jobs/blob/master/centos.org/jobs/candlepin-pipelines.yml
+- Create new GPG key for release
+  - [ ] VERSION=<%= release %> PROJECT=candlepin <%= rel_eng_script('generate_gpg') %>
+  - [ ] VERSION=<%= release %> PROJECT=candlepin <%= rel_eng_script('export_gpg_private') %> to back up
+  - [ ] VERSION=<%= release %> PROJECT=candlepin <%= rel_eng_script('sign_gpg') %>
+  - [ ] VERSION=<%= release %> PROJECT=candlepin <%= rel_eng_script('upload_gpg') %>
```

Comment:
I don't think this renders to proper copy paste commands, because this is the result:
```
- Create new GPG key for release
  - [ ] VERSION=4.5 PROJECT=candlepin [generate_gpg](https://github.com/theforeman/theforeman-rel-eng/blob/master/generate_gpg)
  - [ ] VERSION=4.5 PROJECT=candlepin [export_gpg_private](https://github.com/theforeman/theforeman-rel-eng/blob/master/export_gpg_private) to back up
  - [ ] VERSION=4.5 PROJECT=candlepin [sign_gpg](https://github.com/theforeman/theforeman-rel-eng/blob/master/sign_gpg)
  - [ ] VERSION=4.5 PROJECT=candlepin [upload_gpg](https://github.com/theforeman/theforeman-rel-eng/blob/master/upload_gpg)
```

In other procedures we would generate `VERSION=4.5 PROJECT=candlepin ./generate_gpg` (including backticks to render as code) which can be pasted in the terminal.